universal-login-relayer/lib/utils/ENS/DomainRegistrar.js

```diff
@@ -34,7 +34,7 @@ class DomainRegistrar extends ENSRegistrarBase {
     this.log(`${label}.${tld} owner set to: ${await this.ens.owner(node)} (registrar)`);
   }
 
-  async start(label, tld) {
+  async start(label, tld, save = saveVariables) {
```

Comment:
Let's split that into two functions: `register` and `registerAndSafe`.
We can then avoid ugly callback.

Reply:
✅